src/Akka.Persistence.MongoDb.Hosting/README.md

```diff
@@ -0,0 +1,41 @@
+# Akka.Persistence.MongoDb.Hosting
```

Comment:
Some input if this is sufficient for a readme would be appreciated. Is there anything additional that would be useful to be covered.